package-overrides/github/sytelus/cryptojs@3.1.2.json

```diff
@@ -0,0 +1,129 @@
+{
+  "main": "core.js",
+  "format": "global",
+  "directories": {
+    "lib": "components"
+  },
+  "shim": {
+    "core.js": {
+      "exports": "CryptoJS"
+    },
+    "sha224.js": {
+      "deps": [
+        "cryptojs/sha256.js"
+      ]
```

Comment:
I see.

It often extend CryptoJS and Crypto.algo; e.g. `sha512.js` adds `CryptoJS.SHA512` and `CryptoJS.algo.SHA512`. The former is a helper function to create the hash directly, the later is an object that other Crypto component will need (eg., you would pass it the pbkdf2 function to define which hasher it should use).

Would it work with something that?
```
    "sha512.js": {
      "deps": [
        "cryptojs/x64-core.js"
      ]
      "exports": {
         "hasher": "CryptoJS.SHA512",
         "algo": "CryptoJS.algo.SHA512"
      }
    },
```

Then could it be imported like that?
```
import {hasher, algo} from 'cryptojs/sha512.js';
```